lib/ansible/modules/cloud/docker/docker_login.py

```diff
@@ -22,9 +22,10 @@
 version_added: "2.0"
 description:
     - Provides functionality similar to the "docker login" command.
-    - Authenticate with a docker registry and add the credentials to your local Docker config file. Adding the
-      credentials to the config files allows future connections to the registry using tools such as Ansible's Docker
-      modules, the Docker CLI and Docker SDK for Python without needing to provide credentials.
+    - Authenticate with a docker registry and add the credentials to your local Docker config file respectively the
+      credentials store associated to the registry. Adding the credentials to the config files resp. the credential
```

Comment:
i don't use it, so only a small suggestion and question about the doc
```suggestion
      credentials store associated to the registry.
    - Adding the credentials to the config files resp. the credential
```
and ```resp. the``` is this an abbreviation?

Reply:
I would keep it in one paragraph, as these two sentences belong together.

And `resp.` is an abbrevation for `respectively`.

Reply:
Ok, thanks for clarifying! will know